decidim-elections/app/views/decidim/elections/admin/elections/_form.html.erb

```diff
@@ -0,0 +1,104 @@
+                              button_class: "button button__sm button__transparent-secondary",
+                              help_i18n_scope: "decidim.forms.file_help.image",
+                              help_text: t("decidim.elections.admin.elections.form.attachment_legend") %>
+        </div>
+      </div>
+    </div>
+
+    <div class="card form-calendar" data-component="accordion" id="accordion-calendar">
+      <div class="row column">
+        <button type="button" class="card-divider-button" data-controls="panel-calendar" data-open="true">
+          <%= icon "arrow-right-s-line" %>
+          <h2 class="card-title">
+            <%= t("calendar", scope: "decidim.elections.admin.elections.form") %>
+          </h2>
+        </button>
+      </div>
+      <div id="panel-calendar">
+        <div class="row column">
+          <%= cell "decidim/announcement", t("decidim.elections.admin.elections.form.calendar_description",
+                                             timezone: "#{current_organization.time_zone} (#{l(Time.current.in_time_zone(current_organization.time_zone), format: :long)})"
+          ), callout_class: "warning" %>
+        </div>
+        <div class="row column">
+          <%= form.check_box :manual_start, label: t("decidim.elections.admin.elections.form.manual_start") %>
+        </div>
+        <div class="row column election-fields--time">
+          <div class="md:flex w-100">
+            <div class="row column election_start_time">
+              <%= form.datetime_field :start_at, label: t("decidim.elections.admin.elections.form.start_at") %>
+            </div>
+            <div class="row column">
+              <%= form.datetime_field :end_at, label: t("decidim.elections.admin.elections.form.end_at") %>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="card form-results" data-component="accordion" id="accordion-results">
+      <div class="row column">
+        <button type="button" class="card-divider-button" data-controls="panel-results" data-open="true">
+          <%= icon "arrow-right-s-line" %>
+          <h2 class="card-title">
+            <%= t("results_availability.title", scope: "decidim.elections.admin.elections.form") %>
+          </h2>
+        </button>
+      </div>
+      <div id="panel-results">
+        <div class="row column">
+          <%= form.collection_radio_buttons :results_availability,
+                                            [
+                                              ["real_time", t("decidim.elections.admin.elections.form.results_availability.real_time")],
+                                              ["per_question", t("decidim.elections.admin.elections.form.results_availability.per_question")],
+                                              ["after_end", t("decidim.elections.admin.elections.form.results_availability.after_end")]
+                                            ],
```

Comment:
@antopalidi, for this, you could implement a `results_availability_labels` method in your form class, then  use something like: 
```suggestion
          <%= form.collection_radio_buttons :results_availability,
                                            form.results_availability_labels,
```

while the method could be: defined as 

def type_of_meeting_select
  Decidim::Elections::Election::RESULTS_AVAILABILITY_OPTIONS.map do |type|
    [type, I18n.t("decidim.elections.admin.elections.form.results_availability.#{type}")],
  end
end